### DISCUSSION TITLE
eSIM support...?

### DISCUSSION THREAD

**Discussion by arsradu** [kyujin-cho/pixel-volte-patch#7] [Q&A] [answered]
Hi guys!

This is only a question. And NO, the purpose is not to push you in any way. Your speed is honestly incredible as it is.  :))

I just wanted to ask if it's maybe in your plans to bring support for eSIMs, as well? It seems the services are not being enabled for some carriers (Digi RO MCC-MNC: 226-05, for example), if using eSIM in a dual-SIM setup. Could be something specific to this particular setup, but should this be already supported by default? If not, can it be enabled (technically)? I'm asking for a Pixel 6.

Looking forward to your replies! And once again, congratulations! This is awesome! :) Thank you so much!

**Comment by yawor** [comment, marked as answer]
I don't think it matters whether it's a physical SIM or eSIM. For me this patch enabled VoLTE and VoWiFi (also enabled video calls) for eSIM carrier (Orange PL 260-03), but for the one using physical SIM (Plus PL 260-01) didn't work at all. Both carriers support both technologies.
--edit--
I've disabled eSIM, restarted the phone and then I've been able to apply the patch to the physical card in my case. After enabling eSIM and restarting the phone I now have VoLTE and VoWiFi enabled on both cards.
@arsradu in your case you probably just need to remove your physical SIM card, reboot the phone and run the application. You should be able to enable the VoLTE/VoWiFi for eSIM. Then just reinsert the SIM card and reboot the phone.

I've also created a new issue #10 regarding the dual SIM support.

**Reply by VlaDiRO88** [reply]
With the new update is working v1.2.0

**Reply by yawor** [reply]
Yes, it works!

**Comment by arsradu** [comment]
Thank you very much, guys, for your feedback! :) This app is incredible! And the development is lightning fast!

Kudos to the dev(s)!
